.travis.yml

```diff
@@ -18,6 +18,7 @@ before_script:
   - mkdir -p test/_build
 
 script:
+  - cp phpunit.xml.dist phpunit.xml
   - vendor/bin/phpunit -c ./phpunit.xml
```

Comment:
simply using `vendor/bin/phpunit` will suffice. Ii will look for phpunit.xml.dist

> If phpunit.xml or phpunit.xml.dist (in that order) exist in the current working directory and --configuration is not used, the configuration will be automatically read from that file.